contracts/STokensDescriptor.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MPL-2.0
+pragma solidity 0.8.4;
+
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {ISTokensManager} from "@devprotocol/i-s-tokens/contracts/interface/ISTokensManager.sol";
+import {AddressLib} from "@devprotocol/util-contracts/contracts/utils/AddressLib.sol";
+import {Base64} from "@devprotocol/util-contracts/contracts/utils/Base64.sol";
+import {ISTokensDescriptor} from "./ISTokensDescriptor.sol";
+
+contract STokensDescriptor is ISTokensDescriptor {
+	using Base64 for bytes;
+	using AddressLib for address;
+	using Strings for uint256;
+
+	function getTokenURI(ISTokensManager.StakingPosition memory _position)
+		external override
```

Comment:
`external` to `internal`

Reply:
https://github.com/dev-protocol/s-tokens/pull/1#discussion_r693687910

For the same reason.

@aggre